hashicorp/boundary-ui
# Icu 11802 filter targets by project scope

:tickets: [Jira ticket](https://hashicorp.atlassian.net/browse/ICU-11802)

## Description



## Screenshots (if appropriate):

https://github.com/hashicorp/boundary-ui/assets/29386339/fe12c878-2e9b-461a-bce4-b96b9ada00ac

Added handling for the fact that orgs and projects can not have names.
<img width="224" alt="Screenshot 2023-12-06 at 3 04 37 PM" src="https://github.com/hashicorp/boundary-ui/assets/29386339/cb3953ae-a4fd-4f85-8f89-9cb385a92e43">


## How to Test

Build a binary from the llb-client-daemon branch and use that binary in your local dev environment. You will need to move that binary into the ui/desktop/electron-app/cli folder as well. Be sure to start the desktop client using this command: ENABLE_MIRAGE=false BYPASS_CLI_SETUP=true yarn start:desktop

To test this change, you will need to create some new projects. I would recommend creating a project that has no targets in it to verify that it does not show up. You should only see projects that are included in the list of targets.





## Checklist:
